src/content/1.7/scale/webserver.md

```diff
@@ -0,0 +1,169 @@
+
+It usually works through the http (and https) protocol and delivers web pages to the clients (your web browser). 
+
+{{< figure src="../img/architecture.png" title="webserver's function" >}}
+
+An interesting point to mention is that, php being compiled server side, it often is the webserver's job to manage this task before sending html to the web browser.
+
+## Testing your optimizations
+
+There are plenty of ways to validate the gains of those improvements but a very simple one is to go check any page speed website.
+
+It will point the required changes and as you'll see, most of them will be straightforward to implement.
+
+{{% notice tip %}}
+As usual, the surest way to tune your shop is:
+
+- benchmark
+- make a change
+- repeat
+{{% /notice %}}
+
+### Webservers
+
+We'll show you the different ways to implement those options for both nginx and httpd, the two most popular and widespread webservers.
+
+But before we go on, just a few words about them.
+Don't worry, it won't be long before reaching the crunchy part.
+
+#### nginx
+
+It is often said that nginx is faster than httpd, which is not entirely true. Nginx uses an asynchronous mechanism which is very efficient to sustain heavy loads and static assets (such as pictures and scripts) but slightly increases response time for dynamic content (such as, you know php).
+
+#### httpd
+
+Apache's httpd is the usual suspect and the most popular webserver of alls.
+
+Contrary to nginx, it uses threads that directly serves the contents, being locked during the process.
+
+Which means that if the server receives more calls than available threads it won't be able to answer - hence the reputation of being able to handle less traffic than nginx.
+
+### Optimizations
+
+And now to the crunchy part !
+
+#### Enabling compression
+
+As you may be aware of, most of the file transiting from the webserver (with the notable exception of pictures) are text based.
+
+And text files are the most efficient to compress. Though it's disabled by default, enabling compression is a really quick way to reduce transit and speed up your website up to 80%!
+
+##### Nginx configuration
+
+Here is the way to enable gzip compression to your nginx configuration:
+
+```
+  gzip on;
+  gzip_disable msie6;
+  gzip_vary on;
+  gzip_proxied any;
+  gzip_static on;
+  gzip_buffers 16 8k;
+  gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/javascript image/svg+xml text/js;
+  gzip_http_version 1.1;
+  gzip_comp_level 6;
+  gzip_min_length 1100;
+```
+
+##### Httpd configuration
+
+And here is the way to enable gzip compression to your httpd configuration:
+
+First, make sure the deflate module is activated in your httpd configuration file:
+
+```
+LoadModule deflate_module modules/mod_deflate.so
+```
+
+Then, you can add the following lines to your VirtualHost:
```

Comment:
```suggestion
Then, you can add the following lines to your VirtualHost:
```
I believe this can be done in other configuration files, right 🤔 

Remember : when you write doc, some people (junior developers or junior ops) believe What You Say Is the Truth. So if you say "put it in this file", they will think it cannot go somewhere else. You might want to say you suggest putting it in this file as this is the classical way to do it, but mention other possible areas.

Reply:
To be clear, VirtualHost is not really a file but a configuration section (in a file, obviously ^^).
I tried to be more explicit in the new version, but paths and filenames may vary depending on versions and distribution and OS.
So, it's a bit hard to say, for sure: "add this in this file, at this path".
Maybe "supporting" some OS and version would be a solution.